web/src/pages/cohort/AddFromProjectForm.tsx

```diff
@@ -0,0 +1,234 @@
+import React, { useState, useContext } from 'react'
+import { Container, Form } from 'semantic-ui-react'
+import { useLazyQuery } from '@apollo/client'
+import { uniq } from 'lodash'
+
+import { gql } from '../../__generated__/gql'
+import { ThemeContext } from '../../shared/components/ThemeProvider'
+import MuckError from '../../shared/components/MuckError'
+
+import SequencingGroupTable from './SequencingGroupTable'
+import { Project, SequencingGroup } from './types'
+import { FetchSequencingGroupsQueryVariables } from '../../__generated__/graphql'
+
+const GET_SEQUENCING_GROUPS_QUERY = gql(`
+query FetchSequencingGroups(
```

Comment:
@illusional, this is the query to take the backward route to get SGs based on assay meta. Is there a better way to do this because its v. slow?

Reply:
Yes, we shouldn't construct the query on the front end, we should design a list of filters that we potentially want to filter on, eg:

- batch
- sg created after (/ before) certain date
- projects
- hasCram
- hasGVCF

These would be parameters to an endpoint, and that endpoint would return the list of sequencing groups. We get a lot more flexibility with how we query, and not locked to graphql.

Reply:
(This can be graphql endpoint, or could just be filters on the sequencingGroups method of graphql)

Reply:
How do we find the sg creation timestamp? I can't see a column in the database for it

Reply:
These changes have been moved to a stacked PR https://github.com/populationgenomics/metamist/pull/630